### DISCUSSION TITLE
How to adjust longhorn ui and other components minimum cpu and memory request with helm

### DISCUSSION THREAD

**Discussion by yevon** [longhorn/longhorn#8282] [Q&A]
I'm trying to reduce the minimum requirements of cpu and memory reservation for longhorn components for some dev instances, I've seen in the gui settings the reservation for the manager, but what about other components like the ui, snapshotter etc? How can I control cpu reserved for longhorn? I tried to check the templates directly and default values without success.

**Comment by ChanYiLin** [comment]
Sorry, we don't have cpu/memory reservation control for our components because the consumption grows when the numbers of volume, replicas or other objects increase.

**Reply by Alwinius** [reply]
If there is no minimum cpu or memory, it is equal to zero. This means Kubernetes might schedule the pods on nodes with very little resources. Basically every other Helm Chart on the internet supports defining resources. Why is this not possible for the longhorn Helm Chart?

**Reply by vitobotta** [reply]
Wow, I was searching for an answer and I am very surprised. At least configure some decent number that would work in most configurations to avoid the problem highlighted by @Alwinius ?
